tests/index.js

```diff
@@ -4,7 +4,26 @@ const path = require('path');
 
 const rootTag = '.dc-page';
 
+const Hexo = require('hexo');
+
+const hexo = new Hexo(path.resolve(__dirname, '../'),{});
```

Comment:
@bhaskarmelkani This is fine for now... but remember that will prevent us to reuse this code as a library... why not process.cwd()? it's already better because it means that all you need to do is starting the process from the root of your hexo project.